history.rst

```diff
@@ -1,6 +1,12 @@
 x.x.x (xx-xx-xxxx)
 ------------------
 
+0.16.1 (17-06-2020)
```

Comment:
Release version and date should be added only on release. Keep changes under `x.x.x`.

Reply:
:sweat_smile: Thanks for putting up with me, I've just changed it.